README.md

```diff
@@ -1034,6 +1034,75 @@ const MyComponent = (type) => {
 }
 ```
 
+### 7.8 Using spread operator
+
+When creating a component wrapper we can spread the types from our original component. That way the wrapper extends all the props from the original component automatically. Creating a custom type and passing on `React.FC` typescript can give us all the correct props. This is useful to avoid creating a custom interface for our wrapper with missing props from the original component.
```

Comment:
React.FC was forgotten here 😅

```suggestion
When creating a component wrapper we can spread the types from our original component. That way the wrapper extends all the props from the original component automatically. Creating a custom type typescript can give us all the correct props. This is useful to avoid creating a custom interface for our wrapper with missing props from the original component.
```